docs/CHANGELOG.rst

```diff
@@ -23,6 +23,10 @@ Changed
 * Refactor ``amplicon-archive-multi-report``
 * Use resolwebio/dnaseq:1.2.0 docker in filtering_chemut process
 
+Fixed
+-----
+* Fix consensus_subreads input option in subread process
```

Comment:
Please, put `consensus_subreads` in double backticks (``` `` ```) since it is a code identifier and we want Sphinx to format it differently.
When referencing the Subread process, either use Subread (its name) or ``` ``alignment-subread`` ``` (its slug).

Reply:
It is corrected, hope now is ok.